docs/book/how-to/develop-locally/use-config-files-to-develop-locally.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Learn how to use configuration files to manage local development.
```

Comment:
I think the moment i hear this, i think "oh its slow" . maybe the intent of the chapter is rather that you want to set up different "modes" of your pipeline that load smaller data in a local case? And config files are sort of one way to do that but you can also do it in code right?

Reply:
So reframe it to talk more about having different "variants" of your pipeline in dev, and production.. maybe some if conditions that loads a different dataset or something?

Reply:
done